src/operator/nn/mkldnn/mkldnn_base.cc

```diff
@@ -496,11 +494,6 @@ void OpCheck::Run(mxnet::FCompute fn, const nnvm::NodeAttrs &attrs,
                   const std::vector<mxnet::NDArray> &inputs_,
                   const std::vector<mxnet::OpReqType> &req,
                   const std::vector<mxnet::NDArray> &outputs_) {
-  static auto& is_excluded = Op::GetAttr<bool>("TExcludeMKLDNNDebug");
-  if (is_excluded.get(attrs.op, false)) {
-    LOG(WARNING) << attrs.op->name << " not checked. TExcludeMKLDNNDebug flag present";
-    return;
-  }
```

Comment:
Why is this removed?

Reply:
sorry, I didn't mean to remove the code. It was caused by the wrong merge. I'll handle it.